setup/src/cpp/common/scripts/onCreateCommand.sh

```diff
@@ -31,10 +31,7 @@ echo "#######################################################"
  # Do not change or remove
  # PIP_EXTRA_CONFIG
 
-if [ -z "${VELOCITAS_OFFLINE}" ]; then
-    .devcontainer/scripts/upgrade-cli.sh
-fi
-
+.devcontainer/scripts/upgrade-cli.sh
```

Comment:
Removing the "online variant" check:
At least for the offline variant I would expect that the cli version referenced in the `.velocitas.json` and the installed cli version should be identical. Therefore the check in the upgrade-cli.sh should be sufficient,